### ISSUE TITLE
👀

### ISSUE THREAD

**Issue by twitchyliquid64** [mvirkkunen/nand#1]
Zomg are you making a gate-simulating emulator?

**Comment by mvirkkunen** [comment]
Yes, but it only does nand and it's not very efficient. I just wanted to see if I can design a CPU because I've never made any complicated logic circuits. I still need to implement an actually useful instruction set, but designing those is an endless rabbit hole.
